### ISSUE TITLE
lists of links

### ISSUE THREAD

**Issue by saulpw** [steven-tey/dub#293]
I would love to be able to have a named shortlink go to a page with a simple list of links.  This list could just have the url and page title of each link, and clicking on it would go to the url.  So e.g. example.com/ai could go to a list of links about AI.

Since adding and removing links from lists might require some additional effort, for starters `example.com/<tagname>` could just go to the list of links with that tag.

This would be a killer feature for me.

**Comment by steven-tey** [comment]
@saulpw Love this idea! Would this be a public page or an internal one?

**Comment by saulpw** [comment]
Public, please!  I basically want to be curating a list of links on a topic and pass it around like those "awesome" github repos, but with less overhead (and ideally better design).

**Comment by fmerian** [comment]
could one option be the ability to make your `app.dub.co/links` public? (you can archive/hide sensitive links)

**Comment by RokeJulianLockhart** [comment]
Yes! This would finally allow me to move away from https://github.com/singlelink-co/Singlelink/issues/277#issuecomment-1685311585!
